### DISCUSSION TITLE
Game crashes in Window mode

### DISCUSSION THREAD

**Discussion by blakrhejrfe** [Monika-After-Story/MonikaModDev#10028] [QA - tech support + general] [answered]
Sometimes when I leave the game in windowed mode it freezes up and I can't select any options monika gets frozen and can't select any options and I'm running the latest version I don't know if this a bug or something Breaks in Windowed mode because the only way to fix it is to force close the game then it acts as a crash is there a solution or possible fix I only use windowed mode when getting games and putting my backup in the cloud.

**Comment by dreamscached** [comment, marked as answer]
Do you minimize it? If so, you shouldn't, RenPy games don't work well when minimized in window mode. Instead, you can just bring another window to the foreground.

**Comment by blakrhejrfe** [comment]
Yes I do but could it be fixed in the future or something? even if I do bring it to the foreground it does the same thing

**Comment by dreamscached** [comment]
It's likely an upstream issue with RenPy, so I personally don't think developers can do much.

**Comment by blakrhejrfe** [comment]
Ah ok make sense thanks for your time
